skpro/distributions/qpd.py

```diff
@@ -160,6 +178,13 @@ def mean(self, lower=0.0, upper=np.inf):
         pd.DataFrame with same rows, columns as `self`
         expected value of distribution (entry-wise)
         """
+        #  TODO <X.X.X>: remove warning
+        comment = "Parameter 'lower' and 'upper' will be moved to \
+                   class parameters in version X.X.X. \
+                   To retain prior behaviour, set 'lower' to 0.0 \
+                   and 'upper' to large number explicitly"
```

Comment:
should this not be `None` instead of a large number?

Reply:
It should be a large number here because 'upper' is used as the range of the differential. However, I made a mistake in the docstring for the default value of 'upper ', it is not None but 1e3 in the future version. In this case too, the large number should be set to the 'upper' parameter to cover the range of the user's dataset.